### ISSUE TITLE
Incorrect data-aue-resource declarations in example code?

### ISSUE THREAD

**Issue by vcohen-irvinecompany** [AdobeDocs/experience-manager-cloud-service.en#186]
Issue in ./help/implementing/universal-editor/getting-started.md

In the **Example Connection** code, I see:

```
<ul data-aue-resource="urn:aemconnection:/content/example/list" data-aue-type="container">
<li data-aue-resource="urn:aemconnection/content/example/listitem" data-aue-type="component">
...                                     ^
<li data-aue-resource="urn:fcsconnection:/documents/mytext" data-aue-type="component">
...
 <p data-aue-resource="urn:aemconnection/content/example/another-source" data-aue-prop="title" data-aue-type="text">Photographer</p>
                                        ^
```

In the first `data-aue-resource` declaration, the `referenceName` is followed by a colon, then the leading slash of the resource path. The second declaration skips the colon and appends the resource path directly after the `referenceName`.

The second set of declarations have the same inconsistency.

The **Creating Connections** section immediately before the example seems to indicate that the `referenceName` should always be followed by a colon before the resource path: `data-aue-resource="urn:<referenceName>:<resource>"`

So it seems like the code example is missing a couple of colons in key places. If I'm incorrect, and the code example is correct and deliberate, then it would be helpful to have some comments explaining why the `data-aue-resource` declarations aren't consistent with one another.

**Comment by bohnertchris** [comment]
Thanks for pointing this out, @vcohen-irvinecompany. Indeed the paths should be delimited with a colon. There was likely some formatting lost when we moved content to this page. Sorry for the confusion and we will get this fixed.

**Comment by bohnertchris** [comment]
@adobe export issue to Jira project CQDOC

**Comment by bohnertchris** [comment]
This was corrected in commit 6b90588. Thanks again, @vcohen-irvinecompany, for such a clear problem description. The corrections are now live in the published documentation.
